content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -628,6 +628,47 @@ So a webhook response to add that label would be:
 {{% /tab %}}
 {{< /tabs >}}
 
+Starting in v1.19, admission webhooks can optionally return warning messages that are returned to the requesting client.
+Warnings can be sent with allowed or rejected admission responses.
+No "Warning:" prefix should be included in the message.
+Warning messages should describe a problem the client making the API request should correct or be aware of.
```

Comment:
nits: Some possible rewording, capitalization(?):
Starting in Kubernetes v1.19, Admission Webhooks optionally return warning messages to client requests [ in responses to clients].

nit: a bit confusing: `No "Warning:" prefix should be included in the message.`

nit: another possible reword:
Warning messages describe a problem which the client should correct or be aware of.

Reply:
updated in the restructured version suggested in https://github.com/kubernetes/website/pull/21581#discussion_r454635053